src/rules.js

```diff
@@ -186,7 +186,9 @@ inline.punctuation = edit(inline.punctuation).replace(/punctuation/g, inline._pu
 
 // sequences em should skip over [title](link), `code`, <html>
 inline.blockSkip = /\[[^\]]*?\]\([^\)]*?\)|`[^`]*?`|<[^>]*?>/g;
-inline.escapedEmSt = /\\\*|\\_/g;
+// lookbehind is not available on Safari as of version 16
+// inline.escapedEmSt = /(?<=(?:^|[^\\)(?:\\[^])*)\\[*_]/g;
+inline.escapedEmSt = /(?:^|[^\\])(?:\\[^])*\\[*_]/g;
```

Comment:
Wow. Today I learned that `[^]` matches any character including newlines! I have always used `[\s\S]`.

But don't we just want to match `\\` here so the regex should be `/(?:^|[^\\])(?:\\\\)*\\[*_]/g`?

Reply:
Yes, a mistake on my part. Fixed.